cli/src/main/scala/com/lightbend/rp/reactivecli/Main.scala

```diff
@@ -37,6 +37,35 @@ object Main extends LazyLogging {
 
   val parser = InputArgs.parser(CliName, ProgramVersion.current)
 
+  object MinSupportedSbtReactiveApp {
+    val major: Int = 0
+    val minor: Int = 4
+    val minimum = s"$major.$minor.0"
+
+    def parseVersion(version: String): Option[(Int, Int, Int)] = {
+      // Only strings like "1.2.3" are supported, what comes after
+      // doesn't matter so snapshots like "0.1.2-SNAPSHOT" are okay.
+      try {
+        val parts = version.split("-|\\.")
+        Some(parts(0).toInt, parts(1).toInt, parts(2).toInt)
+      } catch {
+        case _: Exception => None
+      }
+    }
+
+    def isVersionValid(version: String): Boolean = {
```

Comment:
Unit test on this would be nice I think - I've raised an issue: https://github.com/lightbend/reactive-cli/issues/92